automation/test.sh

```diff
@@ -40,15 +40,18 @@ if [ -z $TARGET ]; then
   exit 1
 fi
 
+export KUBEVIRT_DEPLOY_CDI=true
 if [[ $TARGET =~ windows.* ]]; then
   echo "picking the default provider for windows tests"
 elif [[ $TARGET =~ cnao ]]; then
   export KUBEVIRT_WITH_CNAO=true
   export KUBEVIRT_PROVIDER=${TARGET/-cnao/}
+  export KUBEVIRT_DEPLOY_CDI=false
```

Comment:
Did you check that no additional tests were skipped when adding this?

Reply:
yes, all is good, the only skipped were on the sig-operator, which does have KUBEVIRT_DEPLOY_CDI=true
so no problem there